src/jdk.compiler/share/classes/com/sun/tools/javac/processing/PrintingProcessor.java

```diff
@@ -55,7 +55,7 @@
  * deletion without notice.</b>
  */
 @SupportedAnnotationTypes("*")
-@SupportedSourceVersion(SourceVersion.RELEASE_25)
+@SupportedSourceVersion(SourceVersion.RELEASE_26)
```

Comment:
This processor should emulate `com.sun.tools.jdeprscan.TraverseProc` and override `getSupportedSourceVersion`.

Reply:
The implementation of getSupportedSourceVersion inherited from AbstractProcessor uses the value of this annotation.

Reply:
May I prepare a quick patch to bring this in line with `TraverseProc` so we can skip this site for future start of release updates?

Reply:
> May I prepare a quick patch to bring this in line with `TraverseProc` so we can skip this site for future start of release updates?

I know it is possible to write getSupportedSourceVersion to return the latest version, but I don't think it is appropriate for this annotation processor to use that idiom since it might actually need to be meaningfully updated with code changes to support the new source level. Explicitly needing to change this annotation is a backstop reminder to do that.